src/lightning/fabric/utilities/warnings.py

```diff
@@ -23,6 +23,15 @@
 _default_format_warning = warnings.formatwarning
 
 
+def disable_possible_user_warnings(module: str = "") -> None:
```

Comment:
Would this work also as a wrapper so you can disable it only for some code sections?

Reply:
One could pass in the module name to ignore as an argument here, like `disable_possible_user_warnings("lightning.pytorch.strategies")` for example. But I guess for granular warning filters, the user should use the `warnings` library directly as I suggested in the docs page.